src/cores/events/CreateForm.jsx

```diff
@@ -0,0 +1,30 @@
+import React from "react";
+import Form from "./Form";
+import * as eventsServices from "../../services/events.services";
+
+
+const CreateForm = () => {
+
+  const create = async (event) => {
+    try {
+      if (event.status === "Activo"){
+        event.status = true;
+      } else {
+        event.status = false;
+      }
+
+      const data = await eventsServices.createEvent(event);
+      return data;
```

Comment:
Entiendo que "event" viene de un estado, al hacer cosas como `event.status = XXXX`, se esta produciendo una mutacion de estado por un medio que no es "setState" de React!, esto puede ocasionar fallos en un futuro, recomiendo copiar el objeto

```suggestion
  const create = async (event) => {
    try {
      const eventData = { ...event };
      if (event.status === "Activo"){
        eventData.status = true;
      } else {
        eventData.status = false;
      }

      const data = await eventsServices.createEvent(eventData);
      return data;
```

O incluso mas directo...

```suggestion
  const create = async (event) => {
    try {
      return await eventsServices.createEvent({
        ...event,
        status: (event.status === 'Activo')
      });
```